include/LeMonADE/updater/UpdaterSimpleConnection.h

```diff
@@ -96,28 +96,31 @@ class UpdaterSimpleConnection:public AbstractUpdater
   //! get conversion of the reaction process
   double getConversion(){return (1.*NReactedSites)/(1.*NReactiveSites);};
   
-private:
-  //! Specialized move to be used for the movement of the monomers 
-  MoveType move;
-
-  //! Specialized move to be used for the connection between reactive monomers
-  ConnectionMoveType connectionMove;
-
-  //! Number of mcs to be executed
-  uint32_t nsteps;
-
-  //! random number generator (seed set in main program)
-  RandomNumberGenerators rng;
-
+protected:
```

Comment:
Do you need this to be protected? Do you inherit from this Updater?

Reply:
It would like to keep it protected for the inheritance as you mentioned...at least I needed that